improver/wxcode/weather_symbols.py

```diff
@@ -426,13 +345,13 @@ def construct_extract_constraint(
                 coordinate name from diagnostic name
 
         Returns:
-            str or list of str:
-                String, or list of strings, encoding iris cube constraints.
+            iris.Constraint or list of iris.Constraint:
+                Each constraint contains one variable.
```

Comment:
Not sure this is true (generically - although it is true of our current trees), nor particularly useful - you could just leave it with the return type here ie:
```
Returns:
    iris.Constraint or list of iris.Constraint
'''
```

Reply:
Fixed.